### ISSUE TITLE
font size 

### ISSUE THREAD

**Issue by vanderalex** [dmMaze/BallonsTranslator#576]
After last update - imporved automatic text layout for western comic - size of font changed sometimes. It changes when I activate the balloon (but don't go into it). It's hard to explain, just try to work with the text yourself. In general, the logic of working with fonts and balloons is strange, I think there is some bug in the code.

**Comment by dmMaze** [comment]
If you find it hard to put into words just record a video, I tried to do some editing nothing weird happened

**Comment by vanderalex** [comment]
> If you find it hard to put into words just record a video, I tried to do some editing nothing weird happened\



https://youtu.be/PrIkFyZ3uTc

**Comment by dmMaze** [comment]
Looks good to me. 
When you are editing a textblock (double click it), the font menu shows its local format, and the title will change to ```Textblock #some number```. If you're not editing anything it is displayed as global format, which will apply to the RUN process.
If you want to fix font size, set it to ```use global setting``` in config panel -> Typsetting.
And I'd personally recommend https://github.com/NickeManarin/ScreenToGif for recording

**Comment by vanderalex** [comment]
But font size in config panel is set to use global setting! Please look more carefully!

**Comment by dmMaze** [comment]
@vanderalex you're right, let me know if [cafd734](https://github.com/dmMaze/BallonsTranslator/commit/cafd7346ed7158a1c35e1cb205b40c58618e8714) fix it

**Comment by bropines** [comment]
Remeber(no russ... Ladno shutka)

> > If you find it hard to put into words just record a video, I tried to do some editing nothing weird happened\
> 
> [youtu.be/PrIkFyZ3uTc](https://youtu.be/PrIkFyZ3uTc)

**Comment by vanderalex** [comment]
> @vanderalex you're right, let me know if [cafd734](https://github.com/dmMaze/BallonsTranslator/commit/cafd7346ed7158a1c35e1cb205b40c58618e8714) fix it

still problems with bold and itallic (even though they are selected in the font setting, when you start (button RUN) the text comes out without them. Bold button is really strange, it doesn't work sometimes. Also, when you save the style, the font size is saving different (with wrong size). If it is necessary, I can upload another video.
